AndroidAnnotations/androidannotations-core/androidannotations/src/main/java/org/androidannotations/holder/EFragmentHolder.java

```diff
@@ -239,7 +239,7 @@ private void setOnDestroyView() {
 		onDestroyViewAfterSuperBlock = body.blockSimple();
 	}
 
-	private JBlock getOnDestroyViewAfterSuperBlock() {
+	public JBlock getOnDestroyViewAfterSuperBlock() {
```

Comment:
This could be private if you move the field creation to the holder as i suggested.